src/main/java/com/iexec/core/task/update/TaskUpdateManager.java

```diff
@@ -106,7 +109,7 @@ void updateTask(String chainTaskId) {
                 initialized2Running(task, chainTask);
                 break;
             case RUNNING:
-                transitionFromRunningState(task, chainTask);
+                transitionFromRunningState(chainTask, task);
```

Comment:
Why did you switch the two parameters?

Reply:
All methods called from this one have the `(chainTask, task)` order.
As data are copied from `chainTask` to `task`, I told myself it could be more maintainable to have the same order while keeping the notion of `source -> target`.